ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ntp.schema.yml

```diff
@@ -0,0 +1,82 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
```

Comment:
Please ensure that we don't loose useful information from the current documentation:
https://avd.sh/en/devel/roles/eos_cli_config_gen/index.html#ntp

Example valid_values for key_type but also some descriptions that could help.